docs/recipes/gulp/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "gulp-lighthouse-recipe",
```

Comment:
Is this supposed to be gone now https://github.com/GoogleChrome/lighthouse/pull/14183?